demisto_sdk/commands/lint/linter.py

```diff
@@ -816,7 +815,8 @@ def _docker_run_pytest(self, test_image: str, keep_container: bool, test_xml: st
                     with open(file=xml_apth, mode='bw') as f:
                         f.write(test_data_xml)  # type: ignore
 
-                if cov:
+                if not no_coverage:
```

Comment:
why not `if cov`? `if not` is a tiny bit harder to read imo

Reply:
That's what I thought as well, but there is a reason for this. `if cov` tells us if the file exists. But if it doesn't and we didn't specify for `no-coverage` we would like to create it.

Reply:
On one hand, I'd assume there may have been a reason for that in the past (is the author/maintainer still available to answer?) - does removing it really save us a significant amount of time/calls?

If it does, let's hear what @tkatzir says